### PR TITLE
changes in header and footer

### PR DESCRIPTION
**Pull request by gurtatiLND**
- change header and footer

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -11,7 +11,9 @@
   </head>
   <body>
     <header class="row">
-      <a href="/" aria-label="Home" style="margin-left:20px;">BTL Home</a>
+      <div class="row_logo">
+        <a href="#home" aria-label="Home"><img id="logo" alt="agency logo" src="logo.png"></a>
```
Comment (bescka) [inline about.html:15]: 💯

File: contact-us/contact.html [hunk 2]
```diff
@@ -4,21 +4,24 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>BTL Agency Website</title>
-    <link href="styles.css" rel="stylesheet"/>
+    <link href="/styles.css" rel="stylesheet"/>
     <link href="contact.css" rel="stylesheet"/>
+    <link href="contact-us/contact.css" rel="stylesheet"/>
```
Comment (bescka) [inline contact-us/contact.html:9]: contact-us/contact.css will give error from relative file pos.

File: logo.png
Comment (bescka) [inline logo.png:1]: 💯 🥇

**Review by bescka — CHANGES_REQUESTED** [review]

File: contact-us/contact.html (hunk 2 above)
Comment (gurtatiLND) [reply contact-us/contact.html]: yes, I've done

**Review by bescka — APPROVED** [review]

**Pull request merged**